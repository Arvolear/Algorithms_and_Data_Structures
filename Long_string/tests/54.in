23528045900147296518213110408202755781373
183591021246376976379054755249079528