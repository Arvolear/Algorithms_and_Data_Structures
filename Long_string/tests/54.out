23528229491168542895189489462958004860901
23527862309126050141236731353447506701845
4319537974739675027154396586329788288037689766916007394968205816938658031944
128154